test/test_util.py

```diff
@@ -146,13 +194,13 @@ def test_makeDelayFileNames():
         (
             "dir/model_name_wet_01_02_03_ztd.h5",
             "dir/model_name_hydro_01_02_03_ztd.h5"
-        )
+    )
```

Comment:
It was good before.

Reply:
@Askaholic Not sure about this one and the other formatting issues; they ought to be PEP8 compliant (other than line length) as I ran that before the most recent commit (sorry wasn't quite finished with the last push).

Reply:
I think the newest update resolved this.

Reply:
PEP compliance is one thing, beautiful code is another. Ultimately, you want both.

Reply:
For myself I think the current one looks better...